### DISCUSSION TITLE
Re-Locating Video

### DISCUSSION THREAD

**Discussion by jdycus2558** [smplayer-dev/smplayer#57] [General]
I use SMPlayer to play vertically oriented (portrait) mp4 1080P video files on a laptop. I want to display both video and playlist. I can move the playlist over to right of screen, but video is fixed in middle of screen. Is there any way I can move video more to left of screen? Thank you.

**Comment by smplayer-dev** [comment]
Can you send a screenshot?

**Reply by jdycus2558** [reply]
Attached
![Clipboard01](https://user-images.githubusercontent.com/82966373/115711654-c946e380-a341-11eb-8ddd-8a727458efdb.jpg)

**Reply by smplayer-dev** [reply]
You can enable the option "the playlist window is dockable" in preferences -> playlist.

**Reply by jdycus2558** [reply]
Thank you for reply, but it is not the playlist I want to move. I want to move the video (movie) more to the left.

**Reply by smplayer-dev** [reply]
When the playlist is docked it won't over the video, the video will be on the left.
